packages/blockchain-wallet-v4/src/redux/settings/sagas.js

```diff
@@ -193,6 +185,26 @@ export default ({ api }) => {
     yield put(actions.setYubikey())
   }
 
+  const setNotificationsOn = function * ({ enabled }) {
+    const value = enabled ? 2 : 0
+    const guid = yield select(wS.getGuid)
+    const sharedKey = yield select(wS.getSharedKey)
+    const response = yield call(api.enableNotifications, guid, sharedKey, value)
+    if (!contains('updated', response)) { throw new Error(response) }
+    yield put(actions.setNotificationsOn(value))
+  }
+
+  const setNotificationsType = function * ({ types }) {
+    let type = 0
+    if (prop('email', types)) { type = type + 1 }
+    if (prop('mobile', types)) { type = type + 32 }
```

Comment:
What do these magic numbers 1 and 32 mean?

Reply:
Shouldn't it be 2 instead of 32 btw?

Reply:
Nope, basically its a byte system: 1byte 91) is email, 10000bytes (32) for mobile, so you can use 10001 (33) if you want to activate both